db/seeds.rb

```diff
@@ -5,3 +5,8 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+Category.create(name: "Men's")
+Category.create(name: "women's")
+Product.create(sub_category_id:1, name: "Nike Retro", description: "Created for the hardwood but taken verlays and original.", price: 120.5)
+SubCategory.create(category_id: 1, name: "Shoes", description: "80s b-ball icon returns with perfectly")
+Order.create(email: "imran.akhtar@gmail.com", customer_name: "imran", phone_number: 03340053857)
```

Comment:
Use variables and use id through variable instead of static id 
Also, create around 5 sub categories and 10 products here.